ci-operator/step-registry/telcov10n/sno-spoke/site-config/deploy/telcov10n-sno-spoke-site-config-deploy-commands.sh

```diff
@@ -119,6 +119,10 @@ function generate_site_config {
 
     generate_network_config ${baremetal_iface} ${ipi_disabled_ifaces}
 
+    if [ "${root_device}" != "" ]; then
+      ignition_config_override="'{\\\"ignition\\\":{\\\"version\\\":\\\"3.2.0\\\"},\\\"storage\\\":{\\\"disks\\\":[{\\\"device\\\":\\\"${root_device}\\\",\\\"wipeTable\\\":true, \\\"partitions\\\": []}]}}'"
```

Comment:
Do we need 3 backslashes here or only one?

Reply:
Indeed. This was the only way to get the expected format in the Siteconfig.